app/src/main/java/vn/mbm/phimp/me/newGallery.java

```diff
@@ -3698,7 +3729,7 @@ public boolean onLongClick(View v) {
 		ImageView imageSelector;
 	}
 
-	public static void resumeLocalPhoto(){
+	public void resumeLocalPhoto(){
```

Comment:
Aren't you calling this function, in the onScroll function without creating an object shouldn't this be static?